gulpfile.js

```diff
@@ -30,8 +30,8 @@ var path = {
 };
 
 var devServer = {
-  host: '0.0.0.0',
-  port: 4444,
+  host: 'localhost',
```

Comment:
Do not change this.

Feel free to update locally but do not commit this change.